packages/react/src/PatientSummary/Medications.tsx

```diff
@@ -80,9 +80,11 @@ export function Medications(props: MedicationsProps): JSX.Element {
           <Stack h={275}>
             <CodeableConceptInput
               name="request"
+              path="MedicationRequest.medication[x]"
```

Comment:
I think in an ideal world the user wouldn't need to know that the element is choice type or not: would it be possible to use the element name (e.g. `MedicationRequest.medication`) here instead?

Reply:
It's important that `path` property matches the paths specified in a StructureDefinition (and the keys of`InternalTypeSchema.elements`). The utility from this is so that `ElementsContext.getModifiedNestedElement` and can be used as is being done in [HumanNameInput](https://github.com/medplum/medplum/blob/6e353e56ae8dca2581b8c6ec09f2135d6065c0a6/packages/react/src/IdentifierInput/IdentifierInput.tsx#L16) without necessarily knowing the chosen type.